aws.env

```diff
@@ -0,0 +1,7 @@
+NODE_ENV="aws"
+
+# Google Geocode API
+GOOGLE_MAPS_KEY='AIzaSyBj1aLo0X1uH7CKoi3bXPFPdX6eeCra4e8'
```

Comment:
Why did you want to push this API key to the repo? This is a sensitive value

> I encourage you to remove it as quickly as possible from your google console before someone malicious start using it causing undexpected charges on your credit card

Reply:
Well you are right that its a sensitive info and the only motivation behind adding this was just to let reviewer instantly test the functionality(locally) without configuring its own key. To be on safe side, I already have a cap on budget of 5$ monthly to avoid any surprise situation.